### DISCUSSION TITLE
@quasar/app-vite2.0.1. Vue Devtools is detecting ssr mode in "development mode" even though you've built it in production mode

### DISCUSSION THREAD

**Discussion by bekaku** [quasarframework/quasar#17702] [CLI - SSR mode] [answered]
When I use quasar build --mode ssr command and when I run npm start at path ./dist/ssr
I found that Vue Devtools in browser detects that it is development mode and vue tab in inspect can be accessed as in development mode. Not sure where to set it.

I tried console.log('Is Development:', process.env.NODE_ENV !== 'production'); in production, it gives 'Is Development: false'.

I use @quasar/app-vite version 2.0.1.

**Comment by rstoenescu** [comment, marked as answer]
There seems to be a bug in Vue itself. It doesn't care if the flag for it (__VUE_PROD_DEVTOOLS__) is set to Boolean false or true. If it's defined, it's being used. Will report this to the Vue team and also add a workaround in our CLIs.

Edit: reported it [here](https://github.com/vuejs/core/issues/12549)
Edit 2: temporary workaround (until it gets fixed upstream) included in q/app-vite v2.0.2 (just released it)

**Reply by bekaku** [reply]
Thanks
